utils/aws/s3.ts

```diff
@@ -0,0 +1,41 @@
+import aws from "aws-sdk";
+import fs from "fs";
+import path from "path";
+
+const S3 = new aws.S3();
+
+/* 
+
+Uploads a file to the AWS S3 bucket corresponding to AWS_BUCKET_NAME in .env.
+    * filePath: the path of the file to upload
+    * returns: upload response if successful, error object otherwise
+
+Sample usage:
+    myFile = "~/documents/my_file.txt"
+    res = uploadToS3(myFile)
+
+*/
```

Comment:
Use jsdoc syntax for function documentation - see [here](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html)